README.md

```diff
@@ -138,47 +129,50 @@ To run go-spacemesh you need to specify the parameters shared between all instan
 You specify these parameters by providing go-spacemesh with a json config file. Other CLI flags control local node behavior and override default values.
 
 #### Joining a Testnet (without mining)
+
 1. Build go-spacemesh from source code.
-2. Download the testnet's json config file. Make sure your local config file suffix is .json.
+2. Download the testnet's json config file. Make sure your local config file suffix is `.json`.
 3. Start go-spacemesh with the following arguments:
 
-```bash
-./go-spacemesh --tcp-port [a_port] --config [configFileLocation] -d [nodeDataFilesPath]
```

Comment:
what's the current way to set custom ports?
maybe we should mention here?

Reply:
I assume it is `--listen` which takes a sting of the form `/ip4/0.0.0.0/tcp/7513`, at least I didn't find another way.